eZ/Publish/Core/MVC/Symfony/Security/Authentication/RepositoryAuthenticationProvider.php

```diff
@@ -60,4 +63,16 @@ protected function checkAuthentication(UserInterface $user, UsernamePasswordToke
         // Finally inject current user in the Repository
         $this->permissionResolver->setCurrentUserReference($apiUser);
     }
+
+    /**
+     * @throws PasswordInUnsupportedFormatException
```

Comment:
Please use FQCN when you are referencing classes in doc block comments :wink:

Reply:
fixed